### PR TITLE
Radhakrishnan | MOBN-2467 |  Add Monitoring and Performance Tools: Metrics Exporters and Glowroot for OpenMRS

### PR DESCRIPTION
**Pull request by Srkrishna**
https://msfprojects.atlassian.net/browse/MOBN-2467

**Description**
Add metric exporter for Prometheus and integrate Glowroot to monitor OpenMRS API performance. 

Refer to this wiki page for Local setup - https://msfprojects.atlassian.net/wiki/spaces/BahmniLite/pages/1951694857/Grafana+and+Prometheus+Local+Setup+for+Bahmni+Lite

**Screenshots:**
Screenshot 2024-08-22 at 6 43 50 PM
Screenshot 2024-08-22 at 6 43 27 PM
Screenshot 2024-08-22 at 6 43 16 PM
Screenshot 2024-08-22 at 6 43 01 PM

Screenshot 2024-08-22 at 6 44 43 PM

### CODE REVIEW

File: bahmni-lite/docker-compose.yml [hunk 1]
```diff
@@ -62,7 +62,8 @@ services:
       MAIL_PASSWORD: ${MAIL_PASSWORD}
       OMRS_DOCKER_ENV: ${OPENMRS_DOCKER_ENV}
       OMRS_C3P0_MAX_SIZE: ${OMRS_C3P0_MAX_SIZE}
-    #ports:
+    # ports:
```
Comment (supriyatw) [inline bahmni-lite/docker-compose.yml:65]: Why is this commented

File: glowroot/lib/glowroot-central-collector-https-linux-alpine.jar
Comment (supriyatw) [inline glowroot/lib/glowroot-central-collector-https-linux-alpine.jar:1]: Why do we need this lib files also if needed do we need a window jar`glowroot/lib/glowroot-central-collector-https-windows.jar`

File: bahmni-lite/docker-compose.yml (hunk 1 above)
Comment (Srkrishna) [reply bahmni-lite/docker-compose.yml:65]: The idea was to run Glowroot only when needed, so I don't see any benefit in exposing port 4000 when it's not in use. Therefore, I've commented it out.

File: glowroot/lib/glowroot-central-collector-https-linux-alpine.jar
Comment (Srkrishna) [reply glowroot/lib/glowroot-central-collector-https-linux-alpine.jar:1]: Those jar files are part of the Glowroot package itself, Removed it as we don't require them

**Review by supriyatw — APPROVED** [review]

**Pull request merged**